ci/run-k8s-e2e-tests.sh

```diff
@@ -53,6 +56,7 @@ least one failed) and 2 (internal error when running tests, not a test failure).
         --e2e-conformance                                         Run Conformance tests.
         --e2e-whole-conformance                                   Run whole Conformance tests.
         --e2e-network-policy                                      Run Network Policy tests.
+        --e2e-signetwork                                          Run Signetwork tests.
```

Comment:
I never saw it's written as signetwork, I think there is always a dash, can we align with upstream writing?

Reply:
Updated.